docs/src/styles/docs/main.scss

```diff
@@ -48,3 +48,7 @@
     margin-block-start: var(--amplify-space-medium);
   }
 }
+
+.docs-page-tab h2:first-of-type, .docs-meta + h2 {
```

Comment:
This h2 margin is my arch nemesis as I keep finding places where it needs to be tweaked, but this fixes instances like this h2 where it lost it's margin: https://ui.docs.amplify.aws/react/getting-started/introduction#connected-components

Reply:
Would this work (in base.scss):

```scss
h2[id] {
  margin-top: var(--amplify-space-xxxl);
  &:first-child { margin-top: 0; }
}
```

?